eslintrc.json

```diff
@@ -2,114 +2,115 @@
   "extends": "standard",
   "rules": {
     "array-bracket-spacing": [
-      2,
+      "error",
       "never",
       {
       }
     ],
     "brace-style": [
-      2,
+      "error",
       "1tbs",
       {
         "allowSingleLine": true
       }
     ],
     "camelcase": [
-      2,
+      "error",
       {
         "properties": "never"
       }
     ],
     "comma-dangle": [
-      2,
+      "error",
       "never"
     ],
     "comma-style": [
-      2,
+      "error",
       "last"
     ],
     "curly": [
-      2,
+      "error",
       "all"
     ],
-    "dot-notation": 2,
-    "eol-last": 2,
-    "eqeqeq": 2,
-    "guard-for-in": 2,
+    "dot-notation": "error",
+    "eol-last": "error",
+    "eqeqeq": "error",
+    "guard-for-in": "error",
     "indent": [
-      2,
+      "error",
       2,
       {
         "SwitchCase": 1
       }
     ],
     "key-spacing": [
-      2,
+      "error",
       {
         "afterColon": true,
         "beforeColon": false
       }
     ],
     "keyword-spacing": [
-      2,
+      "error",
       {
       }
     ],
-    "linebreak-style": 2,
-    "no-bitwise": 2,
-    "no-caller": 2,
+    "linebreak-style": "error",
+    "no-bitwise": "error",
+    "no-caller": "error",
     "no-empty": [
-      2,
+      "error",
       {
         "allowEmptyCatch": true
       }
     ],
-    "no-extend-native": 2,
-    "no-irregular-whitespace": 2,
-    "no-mixed-spaces-and-tabs": 2,
-    "no-shadow": 2,
-    "no-trailing-spaces": 2,
-    "no-undef": 2,
-    "no-unused-vars": 2,
-    "no-use-before-define": 0,
-    "no-with": 2,
+    "no-extend-native": "error",
+    "no-irregular-whitespace": "error",
+    "no-mixed-spaces-and-tabs": "error",
+    "no-shadow": "error",
+    "no-trailing-spaces": "error",
+    "no-undef": "error",
+    "no-unused-vars": "error",
+    "no-use-before-define": "off",
```

Comment:
I'd rather turn this one on personally, especially as we use more ES6 let and const that aren't hoisted.